mod/wikirate/lib/stylesheets/style_bars_and_boxes.scss

```diff
@@ -1,20 +1,67 @@
 // defaults
-.bar {
-  border-left-color: $border-color !important;
-  &:hover{
-    border-left-color: $dark-text-color !important;
+
+$bar_stripe_color: $gray-700;
+$box_stripe_color: $gray-700;
+
+/*----------- Bars and Boxes -------------*/
+
+.bar, .box {
+  .missing-view {
+    // TODO: obviate in ruby with better unknown API
+    display: none;
   }
 }
 
+.item-box {
+  display: inline-block;
+  min-width: 260px;
+  width: 15%;
+  padding: 10px 1%;
+  vertical-align: top;
+}
+
+.box-top {
+    display: flex;
+    height: 60px;
+    flex-direction: column;
+    justify-content: center;
+    overflow: auto;
+}
+
+.box-middle {
+    height: 180px;
+    img {
+        object-fit: contain;
+        height: 130px;
+        width: 190px;
+    }
+}
+
```

Comment:
Is there any of this that should be in decko?

Reply:
No. I did make some changes to decko, there is PR for that.